src/react/components/common/jsonSchemaFormHelper.ts

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+function backFillAriaLabelledBy(id: string) {
+    const element = document.getElementById(id);
+    if (element) {
```

Comment:
We can merge the if statements
if (element && !element.hasAttribute("aria-labelledby"))

Reply:
Shall we also check if ${id}_erros element exists?